.github/workflows/run_full_test_suite.yml

```diff
@@ -0,0 +1,37 @@
+name: Django Test Suite on PR
+
+on:
+  pull_request:
+    branches:
+      - dev
+
+jobs:
+  run-tests:
+    runs-on: ubuntu-latest
+
+    services:
+      docker:
+        image: docker:19.03.12
```

Comment:
double check this docker image version and make sure we have alignment, for now, do 24.0.5